.tekton/.currency/currency-rbac.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: currency-serviceaccount
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: currency-clusterrole
+rules:
+- apiGroups: [""]
+  resources: ["pods", "pods/log"]
+  verbs: ["get", "list", "watch"]
```

Comment:
Do we really make use of the `watch` verb? If not, then we should go with the principle of the least privilege.